scripts/download_data.sh

```diff
@@ -161,6 +161,78 @@ then
     done
     rm -rf "$(dirname "$folder")"
   done
+elif [ "$DATASET_NAME" = "cf" ]
+then
+  echo "Downloading dataset $1"
+  if [ -d "$DATA_PATH" ]
+  then
+    echo "$DATA_PATH exists."
+  else
+    if [ ! -f "$DATA_DIR/cf.zip" ]
+    then
+      aws s3 cp s3://datasets.ml.labs.aws.intellij.net/codeforces-code-clone/anti-plagiarism-datasets-master.zip "$DATA_DIR/cf.zip"
+    fi
+
+    echo "Unzip dataset"
+    unzip "$DATA_DIR/cf.zip" -d $DATA_DIR/
+    mv "$DATA_DIR"/anti-plagiarism-datasets-master/rounds "$DATA_PATH"
+    rm -rf "$DATA_DIR"/anti-plagiarism-datasets-master
+
+    if [ $DEV ]
```

Comment:
This zip contains several contests. For dev, we can extract just 2-3 of them. See [this](https://unix.stackexchange.com/questions/59276/how-to-extract-only-a-specific-folder-from-a-zipped-archive-to-a-given-directory)